Sources/WasmKit/ModuleParser.swift

```diff
@@ -126,7 +125,8 @@ func parseModule<Stream: ByteStream>(stream: Stream, features: WasmFeatureSet =
         memoryTypes: module.memories.map { $0.type },
         tables: module.tables
     )
-    let enableAssertDefault = _slowPath(getenv("WASMKIT_ENABLE_ASSERT") != nil)
+    #warning("Get env without foundation")
+    let enableAssertDefault = false // _slowPath(getenv("WASMKIT_ENABLE_ASSERT") != nil)
```

Comment:
@kateinoigakukun any suggestions for working around this? This is the only spot where I haven't been able to find a solution yet that doesn't involve Foundation — afaict SwiftSystem doesn't have support for reading the environment.